exonum_client/crypto.py

```diff
@@ -2,6 +2,7 @@
 
 This module uses libsodium as a backend."""
 from typing import Optional
+import logging
```

Comment:
I think since `crypto` is an small auxiliary module with nothing more than primitives, we shouldn't log anything here.